database/migrator.go

```diff
@@ -155,6 +156,29 @@ func getDriver(logger log.Logger, config DatabaseConfig, opts *migrateOptions) (
 				return nil, nil, err
 			}
 		}
+	} else if config.Postgres != nil {
+		if opts.source == nil {
+			src, err := NewPkgerSource("postgres", false)
```

Comment:
We should be able to remove pkger. We've been using go:embed for quite a while. Can leave it in this PR for now.

Reply:
yeah I'll remove that here soon.